kernel-default.properties

```diff
@@ -233,7 +233,9 @@ keycloak.public-client=true
 keycloak.use-resource-role-mappings=true
 keycloak.verify-token-audience=true
 
+#This is the endpoint use by ui(browser) based applications to redirect to open-id system. This URL should be reachable through the browser.
```

Comment:
This is the endpoint used by UI (browser) based applications to redirect to an open-id system. This URL should be reachable through the browser.